lib/Test/Nginx/Util.pm

```diff
@@ -2499,6 +2502,11 @@ sub use_http2 ($) {
             bail_out("cannot use --- http2 with --- pipelined_requests");
         }
 
+        if (defined $block->sni) {
+            $block->set_value("test_nginx_enabled_http2_sni", 1);
+            return 2;
```

Comment:
The use_http2 function returns a boolean value. It's not a good idea to make it return more complex encodings.

Reply:
Sorry, I'll change it.